pkg/config/analysis/msg/messages.yaml

```diff
@@ -644,3 +644,18 @@ messages:
         type: "[]string"
       - name: namespace
         type: string
+
+  - name: "MultiClusterInconsistentService"
+    code: IST0161
+    level: Warning
+    description: "The service is inconsistent across clusters, maybe mixed mode, or conflict ports..."
```

Comment:
It would be helpful to indicate in the message why its inconsistent

Reply:
Thanks, I'll revise with more details.